index.bs

```diff
@@ -1720,6 +1720,10 @@ To <dfn>find matching sources</dfn> given an [=attribution trigger=] |trigger|:
 1. Return |matchingSources|.
 
 To <dfn noexport>trigger attribution</dfn> given an [=attribution trigger=] |trigger|, run the following steps:
+
+1. If |trigger|'s [=attribution trigger/event-level trigger configurations=]
+    [=set/is empty=] and |trigger|'s
+    [=attribution trigger/aggregatable trigger data=] [=list/is empty=], return.
```

Comment:
Currently in the aggregatable attribution algorithm, it's considered no aggregatable data if neither aggregatable trigger data and aggregatable values is present. We should align?